### PR TITLE
Human detection Opencv

### PR DESCRIPTION
**Pull request by ryuk156**
issue : https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93

About Pull request :
This pr consist of Human detection in an image if a Human is detected in an image the green rectangle will be drawn, it uses pre-trained full-body detect classifier

Demo:

image

### CODE REVIEW

**Comment by ryuk156** [comment]
@ankitdobhal  please review and merge

**Review by disha2sinha — CHANGES_REQUESTED** [review]
This is not human detection code

**Comment by ryuk156** [comment]
ohh ,sry i pushed wrong code ,my bad will change it

**Comment by ryuk156** [comment]
@disha2sinha I Have changed the code review, please

File: Image-Processing/Human_detection/Human_detection.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import cv2
+img_file = "Human.jpg"
+classifier_Human = 'car_dect.xml'  #trained human data  Link:"https://raw.githubusercontent.com/opencv/opencv/master/data/haarcascades/haarcascade_fullbody.xml"
+img=cv2.imread(img_file) #create image reader
+black_and_white = cv2.cvtColor(img,cv2.COLOR_BGR2GRAY) #convert image black and white i.e. grayscale 
+Human_detect = cv2.CascadeClassifier(classifier_Human) #create classifier
+humans = Human_detect.detectMultiScale(black_and_white) #detect cars
+
+for (x,y,w,h) in cars:  ## the above variable will return 4 cordinates i.e height,width,postionx,positiony
```
Comment (disha2sinha) [inline Image-Processing/Human_detection/Human_detection.py]: put space after comma

File: Image-Processing/Human_detection/Human_detection.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import cv2
+img_file = "Human.jpg"
+classifier_Human = 'car_dect.xml'  #trained human data  Link:"https://raw.githubusercontent.com/opencv/opencv/master/data/haarcascades/haarcascade_fullbody.xml"
+img=cv2.imread(img_file) #create image reader
```
Comment (disha2sinha) [inline Image-Processing/Human_detection/Human_detection.py]: put space around =

File: Image-Processing/Human_detection/Human_detection.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+black_and_white = cv2.cvtColor(img,cv2.COLOR_BGR2GRAY) #convert image black and white i.e. grayscale 
+Human_detect = cv2.CascadeClassifier(classifier_Human) #create classifier
+humans = Human_detect.detectMultiScale(black_and_white) #detect cars
+
+for (x,y,w,h) in cars:  ## the above variable will return 4 cordinates i.e height,width,postionx,positiony
+    cv2.rectangle(img,(x,y),(x+w,y+h),(0,255,0),3) ##this loop will create grren rectangle when car is detected
```
Comment (disha2sinha) [inline Image-Processing/Human_detection/Human_detection.py]: put space after comma

**Review by disha2sinha — CHANGES_REQUESTED** [review]

**Comment by ryuk156** [comment]
@disha2sinha  done

**Review by disha2sinha — APPROVED** [review]

**Comment by powerexploit** [comment]
> @disha2sinha done

@ryuk156 resolve conflict .

**Comment by ryuk156** [comment]
> > @disha2sinha done
> 
> @ryuk156 resolve conflict .

@ankitdobhal this branch multiple conflicts can you explain how i can resolve this

**Pull request closed without merging**